docs/source/changelog.rst

```diff
@@ -6,6 +6,30 @@ Jupyter notebook changelog
 A summary of changes in the Jupyter notebook.
 For more detailed information, see `GitHub <https://github.com/jupyter/notebook>`__.
 
+.. _release-4.2.0:
+
+4.2.0
+-----
+
+The main new feature of 4.2 is a new API for enabling and installing extensions.
+Extensions can now be enabled at the system-level, rather than just per-user.
+An API is defined for installing directly from a Python package, as well.
+
+.. seealso::
+
+    - :doc:`./examples/Notebook/rstversions/Distributing Jupyter Extensions as Python Packages`
```

Comment:
@minrk This link will need revision since we are building directly from notebooks and there is no ``rstversions`` directory any longer.